config/_default/menus/menus.en.toml

```diff
@@ -15,7 +15,7 @@
 
 [[main]]
   name = "Tutorials"
-  url = "/tutorials/store-and-retrieve/set-up/"
+  url = "/tutorials/lotus/store-and-retrieve/set-up/"
```

Comment:
If you're changing the location of files within this repo, you **need** to add aliases to each file that is moved so that we don't get any 404 errors.